Template for the new rules

encore9870 opened the pull request:
The previous template is outdated for several reasons:
1.  Steps to completion are now posted separately by the PM
2.  The title, assignee, content tag, category tag, subcategory tag should be provided at their according fields, not in the description
3.  Size of the tag is replaced with an estimated time to completion
4.  Description of "why this is good for SCRF" is now required to be "An explanation of why this summary is novel, timely, and interesting"

Therefore I'm proposing this new issue template to reflect the need for change to the reasons above, and in addition:
5. To remind the users to add their draft to the shared Google folder
6. To remind the users to follow the summary template for their draft
7.  To provide guidance and a reminder to what kind of approval they need from LCR to do a research summary

The template has been reviewed and passed by the moderation team (Rachel) to check for style and grammar.

encore9870 committed:
Template for the new rules

The previous template is outdated for several reasons:
1.  Steps to completion are now posted separately by the PM
2.  The title, assignee, content tag, category tag, subcategory tag should be provided at their according fields, not in the description
3.  Size of the tag is replaced with an estimated time to completion
4.  Description of "why this is good for SCRF" is now required to be "An explanation of why this summary is novel, timely, and interesting"

Therefore I'm proposing this new issue template to reflect the need for change to the reasons above, and in addition:
5. To remind the users to add their draft to the shared Google folder
6. To remind the users to follow the summary template for their draft
7.  To provide guidance and a reminder to what kind of approval they need from LCR to do a research summary

The template has been reviewed and passed by the moderation team (Rachel) to check for style and grammar.

ghettodev commented:
Hi @twan-dev, in regards to the `category tag, subcategory tag should be provided at their according fields` part of your ticket, those refer to the proposed category and tags in the forum, not in GitHub. I.e. The researcher should propose something like "Consensus" as a category and "chain-reorg-attacks" as a possible tag. Has that part of the process been moved somewhere else?

ghettodev committed:
adding link to template

- https://github.com/smartcontractresearchforum/docs/blob/main/en/content_research_summary_template.md
- formatting
